lib/tasks/pipeline.js

```diff
@@ -93,11 +92,29 @@ module.exports = Task.extend({
 
     var pluginNameRegex = /^(ember\-cli\-deploy\-)(.*)$/;
     var name            = addon.name.match(pluginNameRegex)[2];
-    if (configuredPluginList && configuredPluginList.indexOf(name) === -1) {
-      return;
-    }
 
-    var deployPlugin = addon.createDeployPlugin({name: name});
+    var configuredNameRegex = /(.*?):(.*)/;
+    var configuredPluginHash = Object.create(null);
+    (deployConfig.plugins || []).forEach(function (pluginName) {
+      var split = pluginName.split(':');
+      if(split[0] === name) {
+        if(split[1]) {
+          configuredPluginHash[pluginName.match(configuredNameRegex)[2]] = split[0];
```

Comment:
The regex also supports `redis:foo:bar` which `split[1]` does not. If you don't need that feature - I certainly do not either, I just felt it was one thing less to document for users :-)